app/src/controllers/lang/reducer.js

```diff
@@ -0,0 +1,10 @@
+import { CHANGE_LANG_ACTION } from './constants';
+
+export const langReducer = (state = { lang: 'be' }, { type, payload }) => {
```

Comment:
Maybe it's better to have a string as a state on this level? Like `state = 'be'`. So it will be `state.lang`, not `state.lang.lang` as a result.